ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -506,33 +542,63 @@ <h3 class="title">
           <p>{{ element.get('spColName').value }}</p>
         </td>
       </ng-container>
+      <ng-container matColumnDef="spAscOrDesc">
```

Comment:
We should add this column for source as well to keep things consistent

Reply:
Added